pyproject.toml

```diff
@@ -112,6 +112,12 @@ exclude_also = [
 strict = ["starcraft"]
 pythonVersion = "3.8"
 pythonPlatform = "Linux"
+exclude = [
+    "**/.*",
+    "**/__pycache__",
+    # pyright might not like the annotations generated by setuptools_scm
+    "starcraft/_version.py",
```

Comment:
What do you think about using `"**/_version.py` instead, so that downstream projects can use this as-is?

Reply:
Ooh, I like that better!